### PR TITLE
New icons

### PR DESCRIPTION
**Pull request by bastianallgeier**
## Features

- New icons
  - archive
  - badge
  - brush
  - discord
  - discount
  - folder-structure
  - github
  - money
  - palette
  - pen
  - store
  - ticket

Screenshot 2022-06-13 at 17 01 25

## Enhancements

- Improved file icons

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [x] Add changes to release notes draft in Notion
- [ ] ~~Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)~~

### CODE REVIEW

File: panel/src/components/Layout/ItemImage.vue [hunk 1]
```diff
@@ -18,7 +18,6 @@
     <k-aspect-ratio v-else :ratio="ratio">
       <k-icon
         :color="$helper.color(image.color)"
-        :size="size"
```
Comment (distantnative) [inline panel/src/components/Layout/ItemImage.vue:21]: Not supporting `size` here doesn't seem right - the component should keep supporting it.

Comment (bastianallgeier) [reply panel/src/components/Layout/ItemImage.vue:21]: You are right. That was obviously just the most lazy way to solve it :)

Comment (bastianallgeier) [reply panel/src/components/Layout/ItemImage.vue:21]: Oh wait, the icon actually never really supported the size attribute. It's just a computed prop that handles the size according to the layout prop.

Comment (distantnative) [reply panel/src/components/Layout/ItemImage.vue:21]: Huh? It does support a `size` prop: https://github.com/getkirby/kirby/blob/main/panel/src/components/Misc/Icon.vue#L7

Comment (distantnative) [reply panel/src/components/Layout/ItemImage.vue:21]: Ah now get it, in `k-item-image` you mean. Ok then this line removal could stay but we should also remove the computed prop.

Comment (distantnative) [reply panel/src/components/Layout/ItemImage.vue:21]: Although this means that also emojis etc. will be sized differently now - not just your new file icons. Not sure if that's what we want.

Comment (bastianallgeier) [reply panel/src/components/Layout/ItemImage.vue:21]: I'll give it another look tomorrow. But I think it's fine if the emojis are also just the regular size in cards. If someone complains we can still turn it back :)

Comment (bastianallgeier) [reply panel/src/components/Layout/ItemImage.vue:21]: Looks good to me …
Screenshot 2022-06-14 at 10 41 14

Comment (LeBenLeBen) [reply panel/src/components/Layout/ItemImage.vue:21]: I’m going to be the one that says it was better before 😬 It helps distinguish pages better I think.

Capture d’écran 2022-07-14 à 14 46 25

Capture d’écran 2022-07-14 à 14 46 16

**Pull request merged**